src/main/resources/templates/multi-tool.html

```diff
@@ -80,7 +80,6 @@
   <script type="module">
     import PdfContainer from './js/multitool/PdfContainer.js';
     import DragDropManager from "./js/multitool/DragDropManager.js";
-    import scrollDivHorizontally from "./js/multitool/horizontalScroll.js";
```

Comment:
Shall we remove the file altogether if it's unused?

Reply:
File now removed